tools/test.py

```diff
@@ -1364,69 +1364,124 @@ def ReadConfigurationInto(path, sections, defs):
-      default=False, action="store_true")
-  result.add_option("--cat", help="Print the source of the tests",
-      default=False, action="store_true")
+  result.add_option("--arch",
+                    default="none",
+                    help="The architecture to run tests for")
+  result.add_option("--abort-on-timeout",
+                    action="store_true",
+                    default=False,
+                    dest="abort_on_timeout",
+                    help="Send SIGABRT instead of SIGTERM to kill processes"
+                         " that time out")
+  result.add_option("--cat",
+                    action="store_true",
+                    default=False,
+                    help="Print the source of the tests")
+  result.add_option("--check-deopts",
+                    action="store_true",
+                    default=False,
+                    help="Check tests for permanent deoptimizations")
+  result.add_option("--expect-fail",
+                    action="store_true",
+                    default=False,
+                    dest="expect_fail",
+                    help="Expect test cases to fail")
   result.add_option("--flaky-tests",
-      help="Regard tests marked as flaky (run|skip|dontcare)",
-      default="run")
-  result.add_option("--warn-unused", help="Report unused rules",
-      default=False, action="store_true")
-  result.add_option("-j", help="The number of parallel tasks to run",
-      default=1, type="int")
-  result.add_option("-J", help="Run tasks in parallel on all cores",
-      default=False, action="store_true")
-  result.add_option("--time", help="Print timing information after running",
-      default=False, action="store_true")
-  result.add_option("--suppress-dialogs", help="Suppress Windows dialogs for crashing tests",
-        dest="suppress_dialogs", default=True, action="store_true")
-  result.add_option("--no-suppress-dialogs", help="Display Windows dialogs for crashing tests",
-        dest="suppress_dialogs", action="store_false")
-  result.add_option("--shell", help="Path to V8 shell", default="shell")
-  result.add_option("--store-unexpected-output",
-      help="Store the temporary JS files from tests that fails",
-      dest="store_unexpected_output", default=True, action="store_true")
+                    default="run",
+                    help="Regard tests marked as flaky (run|skip|dontcare)")
+  result.add_option("-j",
+                    default=1,
+                    help="The number of parallel tasks to run",
+                    type="int")
+  result.add_option("-J",
+                    action="store_true",
+                    default=False,
+                    help="Run tasks in parallel on all cores")
+  result.add_option("--logfile",
+                    dest="logfile",
+                    help="Write test output to file. NOTE: this only applies"
+                         " the tap progress indicator")
```

Comment:
I know this was here before, but while you're here:

`" to the tap progress indicator"`

Reply:
done